doc/api/buffer.markdown

```diff
@@ -728,7 +728,7 @@ const buf = new Buffer([1,-2,3,4]);
 buf.readInt16BE();
   // returns 510
 buf.readInt16LE(1);
-  // returns -511
+  // returns 1022
```

Comment:
I think the original intent was to read the same two bytes.
cc @jasnell who I believe added this example.